pygfx/objects/_base.py

```diff
@@ -115,28 +120,57 @@ def __init__(self):
 
     @property
     def id(self):
-        """An integer id smaller than 2**31."""
+        """An integer id smaller than 2**31 (read-only)."""
         return self._id
 
+    @property
+    def visible(self):
+        """Wheter is object is rendered or not. Default True."""
+        return self._visible
+
+    @visible.setter
+    def visible(self, visible):
+        self._visible = bool(visible)
+
+    @property
+    def parent(self):
+        """Object's parent in the scene graph (read-only).
+        An object can have at most one parent.
+        """
+        return self._parent
+
     @property
     def children(self):
+        """The child objects of this wold object (read-only tuple).
+        Use ``.add()`` and ``.remove()`` to change this list.
+        """
         return tuple(self._children)
 
-    def add(self, obj):
-        # orphan if needed
-        if obj.parent is not None:
-            obj.parent.remove(obj)
-        # attach to scene graph
-        obj.parent = self
-        self._children.append(obj)
-        # flag world matrix as dirty
-        obj.matrix_world_dirty = True
+    def add(self, *objects):
+        """Adds object as child of this object. Any number of
+        objects may be added. Any current parent on an object passed
+        in here will be removed, since an object can have at most one
+        parent.
+        """
+        for obj in objects:
+            # orphan if needed
+            if obj._parent is not None:
+                obj._parent.remove(obj)
+            # attach to scene graph
+            obj._parent = self
+            self._children.append(obj)
+            # flag world matrix as dirty
+            obj.matrix_world_dirty = True
         return self
 
-    def remove(self, obj):
-        if obj in self._children:
-            obj.parent = None
-            self._children.remove(obj)
+    def remove(self, *objects):
+        """Removes object as child of this object. Any number of objects may be removed.
+        """
+        for obj in objects:
+            if obj in self._children:
+                obj._parent = None
+                self._children.remove(obj)
```

Comment:
Just a micro optimization, but, I think this would be faster:

```
try:
    self._children.remove(obj)
    obj._parent = None
except ValueError:
    pass. # or just let it raise?
```

It would avoid one list traversal :)

Reply:
done